### ISSUE TITLE
[Bug] Vertical "Full Screen"

### ISSUE THREAD

**Issue by KarmaBeast** [arichornlover/uYouEnhanced#531]
When the video is open and you swipe down the player enters vertical mode, and the video orientation becomes really weird. This annoyance happens pretty regularly:

![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/c5e753ed-d2d5-470f-ac00-f1e8d91b7853)

**Comment by KarmaBeast** [comment]
Btw @arichornlover We are currently at 425 Members in the Telegram group. Just a heads up ;)

**Comment by bhackel** [comment]
I am also noticing this bug recently and having trouble replicating it

**Comment by KarmaBeast** [comment]
Do you mean how to reproduce it? You just swipe down anywhere on the YouTube app screen, while the video is open in normal mode

**Comment by bhackel** [comment]
Oh it's specifically on the title/description area, not the player area. I would think that it can be disabled then

**Comment by KarmaBeast** [comment]
Yeah it's only happening when you swipe down anywhere below the video area:

![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/d5799841-b143-4e2b-9071-aea28d10a937)

(Swiping down on the video area itself only enables MiniPlayer which is working as intended)

**Comment by bhackel** [comment]
I found out that it doesn't happen in versions before 19.XX. I will look into the A/B flags soon

**Comment by KarmaBeast** [comment]
Gotchu, it would be nice to fix this YouTube bug!

**Comment by KarmaBeast** [comment]
@bhackel Would it be possible to add a exception to this rule? For real vertical videos. I mean I could still swipe up on those to enter fullscreen, but still keeping swipe down on the small area below would be nice. If this tweak is able to detect real vertical videos. 

Example link: https://youtu.be/Fh_pzNxcfw0?si=NqeHloiAImy8vVgk

https://github.com/arichornlover/uYouEnhanced/assets/73917917/b8640735-ef62-4ed9-a1d2-c3542d361d17

**Comment by bhackel** [comment]
Working on it, will be done in the next couple days

**Comment by KarmaBeast** [comment]
Thanks guys appreciate the work☝🏻

**Comment by KarmaBeast** [comment]
@bhackel In case you managed to exclude vertical videos from this tweak, I'd like to test it! Please let me know so I can try to grab IPA
